src/app/config/url.config.json

```diff
@@ -8,7 +8,13 @@
     "project": {
         "listingUrl": "/project/v1/solutions/targetedSolutions"
     },
-    "homeListing": {
+    "formListing": {
```

Comment:
why are we changing the name from home to form here? are not using in the home page anymore?

Reply:
ok